bitcoin/testing.py

```diff
@@ -27,11 +31,10 @@ def raise_for_status(self):
         pass
 
 
-import requests
 requests.get = lambda *args, **kwargs: FakeResponse()
+requests.request = lambda *args, **kwargs: FakeResponse()
```

Comment:
This wrapper should be a bit more sophisticated, to only return 200 when `method="GET"`, https://requests.readthedocs.io/en/latest/api/#requests.request.